samples/DataSourceWithProcessingAndExtensions/DataGenerator/Jsonbinary.cs

```diff
@@ -0,0 +1,41 @@
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace DataGenerator
+{
+    public static class JsonBinary
+    {
+        public static byte[] ToBinary(object value)
+        {
+            byte[] result = null;
+
+            using (var stream = new MemoryStream())
+            {
+                using (var writer = new BinaryWriter(stream, Encoding.Default, true))
```

Comment:
We should use an explicit encoding, like UTF8, in both methods to ensure we can round trip the data.

See the warning here:
https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.default?view=netcore-3.1